assets/base/i18n/en.json

```diff
@@ -1899,6 +1899,19 @@
   "mobile.channel_list.privateChannel": "Private Channel",
   "mobile.channel_list.publicChannel": "Public Channel",
   "mobile.channel_list.unreads": "UNREADS",
+  "mobile.client_upgrade": "Client Upgrade",
+  "mobile.client_upgrade.can_upgrade_title": "Upgrade Available!",
+  "mobile.client_upgrade.can_upgrade_subtitle": "There's a new upgrade ready for download.",
+  "mobile.client_upgrade.close": "Upgrade Later",
```

Comment:
In the desktop app, we use `Skip this version`, but it seems to be more common on mobile apps to say `Update Later`, `Not Now` or just `Later`. I'm okay just using `Update Later`.